Mathlib/Data/Finset/NAry.lean

```diff
@@ -4,7 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Yaël Dillies
 
 ! This file was ported from Lean 3 source module data.finset.n_ary
-! leanprover-community/mathlib commit 20715f4ac6819ef2453d9e5106ecd086a5dc2a5e
+! leanprover-community/mathlib commit 517cc149e0b515d2893baa376226ed10feb319c7
 ! Please do not edit these lines, except to modify the commit id
 ! if you have ported upstream changes.
 -/
```

Comment:
https://leanprover-community.github.io/mathlib-port-status/file/data/finset/n_ary?range=20715f4ac6819ef2453d9e5106ecd086a5dc2a5e..517cc149e0b515d2893baa376226ed10feb319c7 says you should have added an import here